data/distribs.json

```diff
@@ -52,5 +52,11 @@
         "main_color": "dd4814",
         "website": "http://www.ubuntu.com",
         "doc": "https://doc.ubuntu-fr.org"
+    },
+    "xubuntu": {
+        "name": "Xubuntu",
+        "main_color": "333333",
```

Comment:
For me, the main color of the distribution is a dark blue (see https://xubuntu.org and its logo).